examples/run_mock_client_main.cc

```diff
@@ -0,0 +1,53 @@
+#include <iostream>
+#include <memory>
+#include <string>
+
+#include "grpcpp/grpcpp.h"
+#include "src/common/protocols/protocol_manager.h"
+#include "src/protos/grpc/master_metadata_service.grpc.pb.h"
```

Comment:
nit: no need to include this here. Once someone includes the master_metadata_service_client.h (i.e. protocol_manager.h), it will include the necessary deps.

Reply:
I disagree. We should not depend on another header file's include to import something that we need in our file, especially if the two files and binaries are logically independent. If any one of them  change their implementation and imports, it should not break this file is trying to import "master_metadata_service.grpc.pb.h" to use `OpenFileRequet`.